petergreve/application/test/stores/message-store.js

```diff
@@ -0,0 +1,35 @@
+var rewire = require("rewire");
+var Constants = require('../../app/constants/constants')
+var MessageStore = rewire('../../app/stores/message-store')
+import Dispatcher from '../../app/dispatcher/dispatcher'
+
+describe('Message Store methods', () => {
+	
+	before(() => {
+        MessageStore.__set__('_messages',[])

```

Comment:
I didn't notice you had used `rewire`, and thought you had built the `__set__` function yourself.
I think it is a good approach regarding store state in tests, probably the best so far. 
Good job!